From: Tommy Webb <tommy@calyxinstitute.org>
Date: Tue, 11 Feb 2025 15:33:55 +0000
Subject: Disable additional segmentation platform features

Fix a crash on non-first startup related to home module ranking,
which relies on segmentation platform which is already disabled via
Remove-segmentation-platform.patch. A Chromium fix would be ideal
in concept so that when the core feature (kSegmentationPlatformFeature)
is disabled, its "subfeatures" are not still consulted -- at least,
that would make sense to me. However, it's easy for us to just turn off
these "subfeatures" ourselves for now.

Change-Id: Id37707403644bb83f4ac0df49d7baf86b8b94b81
---
 .../Disable-additional-segmentation-platform-features.inc      | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 cromite_flags/components/segmentation_platform/public/features_cc/Disable-additional-segmentation-platform-features.inc

diff --git a/cromite_flags/components/segmentation_platform/public/features_cc/Disable-additional-segmentation-platform-features.inc b/cromite_flags/components/segmentation_platform/public/features_cc/Disable-additional-segmentation-platform-features.inc
new file mode 100644
--- /dev/null
+++ b/cromite_flags/components/segmentation_platform/public/features_cc/Disable-additional-segmentation-platform-features.inc
@@ -0,0 +1,3 @@
+SET_CROMITE_FEATURE_DISABLED(kSegmentationPlatformAndroidHomeModuleRanker);
+SET_CROMITE_FEATURE_DISABLED(kSegmentationPlatformAndroidHomeModuleRankerV2);
+SET_CROMITE_FEATURE_DISABLED(kSegmentationPlatformEphemeralCardRanker);
-- 

